src/DualQuaternion.jl

```diff
@@ -4,6 +4,14 @@ struct DualQuaternion{T<:Real} <: Number
   q0::Quaternion{T}
   qe::Quaternion{T}
   norm::Bool
+  function DualQuaternion{T}(q0::Quaternion, qe::Quaternion, norm::Bool) where T <: Real
+    Base.depwarn("`DualQuaternion` is deprecated and will be removed in the next breaking release. Use `Quaternion{ForwardDiff.Dual}` instead.", :DualQuaternion)
+    return new{T}(q0, qe, norm)
+  end
+end
+
+function DualQuaternion(q0::Quaternion{T}, qe::Quaternion{T}, norm::Bool) where T <: Real
```

Comment:
Why is this method necessary?

Reply:
Without this method, `DualQuaternion(quat(1,2,3,4), quat(5,6,7,8), false)` throws `MethodError`. (On the current `master`, this doesn't throw errors.)